pages/common/lt.md

```diff
@@ -0,0 +1,25 @@
+# lt
+
+> Localtunnel exposes your localhost to the world for easy testing and sharing! No need to mess with DNS or deploy just to have others test out your changes.
+> Great for working with browser testing tools like browserling or external api callback services like twilio which require a public url for callbacks.
```

Comment:
Wow, this is a long description. Perhaps we could make this more concise?

Ideally in a tldr page we want to be as factual and neutral as possible too. For example, we like to avoid terms like `easy` if it's a relative term.